src/Illuminate/Database/Query/Builder.php

```diff
@@ -53,13 +53,14 @@ class Builder
      * @var array
      */
     public $bindings = [
-        'select' => [],
-        'from'   => [],
-        'join'   => [],
-        'where'  => [],
-        'having' => [],
-        'order'  => [],
-        'union'  => [],
+        'select'     => [],
+        'from'       => [],
+        'join'       => [],
+        'where'      => [],
+        'groupBy'    => [],
+        'having'     => [],
+        'order'      => [],
+        'union'      => [],
```

Comment:
Let's undo the indentations here for these assignments.

https://madewithlove.be/code-style-options-for-cleaner-diffs/